docs/source/package/plugins.rst

```diff
@@ -9,6 +9,7 @@ Plugin Internal Machinery
    section exists primarily for the aid of Sopel’s core development.
 
 .. toctree::
+    :titlesonly:
```

Comment:
This also feels a little terse, I prefer the output of `:maxdepth: 2` here as well

```suggestion
    :maxdepth: 2
```

The output is definitely still busy, but it's 1.5 pages of busy for me instead of the 7 pages that current unstable docs produces.

![image](https://github.com/sopel-irc/sopel/assets/6125322/fab074e4-cce0-456a-880e-b690aa779090)

Reply:
You're probably right, this can afford to be a bit more verbose as it isn't sitting on top of any "real content" and is just a index page for the rest of the plugin machinery docs.